perfrunbook/utilities/measure_and_plot_basic_pmu_counters.py

```diff
@@ -190,13 +204,16 @@ def get_cpu_type():
 
     parser = argparse.ArgumentParser()
     parser.add_argument("--stat", default="ipc", type=str, choices=stat_choices)
+    parser.add_argument("--period", default=1000, type=int)
+    parser.add_argument("--cpu-list", action="store", type=str)
+    parser.add_argument("--no-plot", action="store_true", help="Only save data to CSV file, do not plot")
```

Comment:
Technically, the "Only save data to CSV file" part isn't really correct. This option has no bearing on saving to CSV, and saving to CSV is off by default.